src/App.js

```diff
@@ -1,8 +1,31 @@
 import React from 'react';
 import './App.css';
+import PropTypes from 'prop-types';
+
+const Sum = props => (
```

Comment:
делай деструктуризацию props
```suggestion
const Sum = ({ a, b }) => (
```